Services/UICore/classes/Setup/class.ilCtrlStructureReader.php

```diff
@@ -73,11 +73,16 @@ public function readStructure(
         // plugin path
         $this->plugin_path = $a_plugin_path;
 
+        $read_plugins = false;
+        if ($this->plugin_path != "" && $this->comp_prefix != "") {
+            $read_plugins = true;
+        }
+
```

Comment:
How will this be set for the case "install --no_plugins"? You won't have a plugin_path or comp_prefix, but you do not want to "read_plugins" in this case, right?

In general it would be ok for me so set a $this->read_plugins instead of passing it, even if this not settled in the constructor, which would be a better place to configure the reader instead doing it in readStructure().

Reply:
done